app/Domains/Redis/Jobs/StoreInsightsJob.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace App\Domains\Redis\Jobs;
+
+use App\Data\Models\Insights;
+use App\Data\Models\Metrics;
+use Lucid\Units\Job;
+use Illuminate\Support\Facades\Redis;
+
+class StoreInsightsJob extends Job
+{
+    /**
+     * @var Insights
+     */
+    private Insights $insights;
+
+    /**
+     * Create a new job instance.
+     *
+     * @param Insights $insights
+     */
+    public function __construct(Insights $insights)
+    {
+        $this->insights = $insights;
+    }
+
+    /**
+     * Execute the job.
+     *
+     * @return void
+     * @throws \Exception
+     */
+    public function handle()
+    {
+        $username = $this->insights->account->username;
+        $account = $this->insights->account;
+
+        // Insights
+        $key = "insights:{$this->insights->platform}:{$username}:latest:account";
+        Redis::set($key, json_encode([
```

Comment:
move to a Repository class as mentioned in the training doc.

Reply:
done